src/pages/Main/HotTea/HotTea.js

```diff
@@ -0,0 +1,156 @@
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [selectTab, setSelectTab] = useState('여름맞이 선물');
+
+  const minusArr = () => {
+    const arrPop = data.pop();
+    data.unshift(arrPop);
+  };
+  const minusPx = () => {
+    setTransition('all ease 0.7s');
+    setMovePx(movePx + 240);
+    setIsDisabled(true);
+
+    setTimeout(() => {
+      setTransition('none');
+      minusArr();
+      setMovePx(0);
+      setIsDisabled(false);
+    }, 700);
+  };
+
+  const plusArr = () => {
+    const arrPop = data.shift();
+    data.push(arrPop);
+  };
+
+  const plusPx = () => {
+    setTransition('all ease 0.7s');
+    setMovePx(movePx - 240);
+    setIsDisabled(true);
+
+    setTimeout(() => {
+      setTransition('none');
+      plusArr();
+      setMovePx(0);
+      setIsDisabled(false);
+    }, 700);
+  };
+
+  const updateSelectTab = ({ target }) => {
+    const currentTabName = target.innerHTML;
+    setSelectTab(currentTabName);
+  };
+
+  return (
+    <div className="main-hottea">
+      <div className="main-hottea-title">
+        <h3>오늘은 어떤 차를 마셔볼까요?</h3>
+        <button
+          className={`main-hottea-summer ${
+            selectTab === '여름맞이 선물' ? 'on' : 'off'
+          }`}
+          onClick={summerData}
+          onMouseDown={updateSelectTab}
+        >
+          여름맞이 선물
+        </button>
+        <button
+          className={`main-hottea-summer ${
+            selectTab === '이번 주 인기 제품' ? 'on' : 'off'
+          }`}
+          onClick={weekData}
+          onMouseDown={updateSelectTab}
+        >
+          이번 주 인기 제품
+        </button>
+      </div>
+      <div className="main-hottea-scroll">
+        <div className="hottea-left">
+          <button onClick={minusPx} disabled={isDisabled}>
+            <i className="fa-solid fa-angle-left" />
+          </button>
+        </div>
+
+        <div className="hottea">
+          <ul
+            className="hottea-list"
+            style={{
+              transform: `translateX(${movePx}px)`,
+              transition: `${transition}`,
+            }}
+          >
+            {data.map(({ id, thumbnail_images, title, price, discount }) => {
+              return (
+                <li className="items" key={id}>
+                  <div className="img-box">
+                    <img
+                      src={thumbnail_images[0]}
+                      alt="상품이미지1"
+                      className="items-img"
+                    />
+                    <div className="hover-box">
+                      <img
+                        src={thumbnail_images[1]}
+                        alt="상품이미지2"
+                        className="items-hover-img"
+                      />
+                      <div className="hover-icon">
+                        <i className="fa-solid fa-cart-shopping" />
+                      </div>
+                    </div>
+                  </div>
+                  {!discount ? (
```

Comment:
조건은 truthy한 조건으로 하는 게 덜 헷갈림.
```suggestion
                  {discount ? (
```